modules/apps/headless/headless-builder/headless-builder-test/src/testIntegration/java/com/liferay/headless/builder/model/listener/test/APISortRelevantObjectEntryModelListenerTest.java

```diff
@@ -110,19 +110,7 @@ public void testPostSort() throws Exception {
 			).toString(),
 			JSONCompareMode.LENIENT);
 
-		JSONObject apiApplicationJSONObject = HTTPTestUtil.invokeToJSONObject(
-			JSONUtil.put(
-				"applicationStatus", "published"
-			).put(
-				"baseURL", StringUtil.toLowerCase(RandomTestUtil.randomString())
-			).put(
-				"externalReferenceCode", RandomTestUtil.randomString()
-			).put(
-				"title", RandomTestUtil.randomString()
-			).toString(),
-			"headless-builder/applications", Http.Method.POST);
-
-		JSONObject apiEndpointJSONObject = HTTPTestUtil.invokeToJSONObject(
+		JSONObject jsonObject = HTTPTestUtil.invokeToJSONObject(
```

Comment:
Maybe better to rename it to apiEndpointJSONObject, as it is more clear in order to debug the test

Reply:
Yes, you are right, I think that is the general rule, but it make us to check what we are creating. Bah, leave it as it is